apps/chatbots/forms.py

```diff
@@ -21,7 +23,14 @@ def __init__(self, request, *args, **kwargs):
 
     @transaction.atomic()
     def save(self, commit=True):
-        pipeline = Pipeline.create_pipeline_with_name(self.request.team, self.cleaned_data["name"])
+        team_id = get_current_team().id
```

Comment:
Using `self.request.team.id` is a simpler and more direct approach and doesn't rely on thread locals.